src/Foundation/NSUrl.cs

```diff
@@ -47,6 +47,9 @@ public bool Equals (NSUrl url)
 		// Converts from an NSURL to a System.Uri
 		public static implicit operator Uri (NSUrl url)
 		{
+			if (url == null) {
```

Comment:
```suggestion
			if (url is null) {
```
Lets make tj work a little less.